### PR TITLE
Show invalid resources errors in the event sheet

### PR DESCRIPTION
**Pull request by D8H**
image

### CODE REVIEW

**Comment by ClementPasteau** [comment]
Niiiiiiiiice 🔥

**Review by ClementPasteau — APPROVED** [review]
lgtm ! ✅

File: Core/GDCore/Extensions/Metadata/ValueTypeMetadata.h [hunk 1]
```diff
@@ -178,6 +178,15 @@ class GD_CORE_API ValueTypeMetadata {
     } else if (type == "variable") {
       return parameterType == "objectvar" || parameterType == "globalvar" ||
              parameterType == "scenevar";
+    } else if (type == "resource") {
+      return parameterType == "fontResource" ||
+             parameterType == "soundfile" ||
```
Comment (ClementPasteau) [inline Core/GDCore/Extensions/Metadata/ValueTypeMetadata.h:183]: just a random question, why are those types not CamelCase?

Comment (D8H) [reply Core/GDCore/Extensions/Metadata/ValueTypeMetadata.h:183]: It's legacy. There is a todo somewhere to renamed them "audioResource".

**Pull request merged**